### PR TITLE
Attempt to fix nightly tag

### PR DESCRIPTION
**Pull request by ohm314**
Nightly build fails because the tag is not correctly set. This PR should fix it.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -142,29 +141,32 @@ jobs:
...
         -e NMODL_NIGHTLY_TAG=$TAG \
         'bluebrain/nmodl:wheel' \
         packaging/build_wheels.bash linux $(python.version)
-    condition: and(succeeded(), or(eq(variables.buildWheel, true), eq(variables['python.version'], '3.9')))
+    condition: and(succeeded(), or(eq(variables.buildWheel, True), eq(variables['python.version'], '3.9')))
     displayName: 'Building ManyLinux Wheel'
+  #- task: PublishBuildArtifacts@1
+  #  inputs:
+  #    pathToPublish: '$(Build.SourcesDirectory)/wheelhouse'
+  #  displayName: 'Publish wheel as build artifact'
```
Comment (pramodk) [inline azure-pipelines.yml:163]: I would uncomment this

File: azure-pipelines.yml [hunk 2]
```diff
@@ -2,21 +2,20 @@
 # Azure Pipeline NMODL settings
 #=============================================================================
 variables:
-- name: ReleaseWheelBuild
-  value: false
-- name: NightlyWheelBuild
-  value: false
+  # defined in UI
+  #- ReleaseWheelBuild: False
+  #- NightlyWheelBuild: False
```
Comment (pramodk) [inline azure-pipelines.yml:7]: I think we can remove this

Comment (ohm314) [reply azure-pipelines.yml:7]: OK, I thought I leave it more for documenting, this way we know that they are defined in UI and can be used inside yaml

File: azure-pipelines.yml (hunk 1 above)
Comment (ohm314) [reply azure-pipelines.yml:163]: You want to always produce artefacts?

Comment (pramodk) [reply azure-pipelines.yml]: yes, that's we typically do. Artefacts get deleted automatically after certain time by Azure automatically .

**Review by pramodk — APPROVED** [review]
LGTM

**Pull request merged**